lile.go

```diff
@@ -63,6 +67,12 @@ type Service struct {
 
 // NewService creates a new service with a given name
 func NewService(n string) *Service {
+	once.Do(func() {
```

Comment:
What about just moving this to an init? We actually have a `lile_windows.go` now

Reply:
moving this to init doesn't work since some packages refer this package on start-up.